### PR TITLE
add: python-colorcet

### PR DESCRIPTION
**Pull request by hsgwa**
Signed-off-by: hsgwa <hasegawa.isp@gmail.com>

Please add the following dependency to the rosdep database.

## Package name:

colorcet

## Package Upstream Source:

https://github.com/holoviz/colorcet

## Purpose of using this:

colorcet is a package that provides easy-to-read color maps.
It is useful for visualization of continuous values and color categorization for each of several types.

official document
https://colorcet.holoviz.org/

Distro packaging links:

## Links to Distribution Packages

pip

https://github.com/holoviz/colorcet

### CODE REVIEW

**Review by methylDragon — APPROVED** [review]

File: rosdep/python.yaml [hunk 1]
```diff
@@ -1180,6 +1180,16 @@ python-colorama:
     pip:
       packages: [colorama]
   ubuntu: [python-colorama]
+python-colorcet:
```
Comment (ivanpauno) [inline rosdep/python.yaml:1183]: You probably want to make it available for python3, in which case it should be:

```suggestion
python3-colorcet:
```

**Review by ivanpauno — CHANGES_REQUESTED** [review]
There're an [ubuntu](https://packages.ubuntu.com/jammy/python/python3-colorcet) and a [debian](https://packages.debian.org/testing/python/python3-colorcet) packages for this.
So we should use those instead, and the pip packages only in the platforms that don't provide it in their nativa package manager.

Please, read the contributing guidelines for python packages https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#python-rules.

File: rosdep/python.yaml (hunk 1 above)
Comment (hsgwa) [reply rosdep/python.yaml]: sorry, I overlooked the description. 
I modified in [502c620](https://github.com/ros/rosdistro/pull/33681/commits/502c620d26d998c05d665ba2fa947fa3df345b52).

File: rosdep/python.yaml [hunk 2]
```diff
@@ -6181,6 +6181,15 @@ python3-colorama:
...
   ubuntu: [python3-colorama]
+python3-colorcet:
+  debian:
+    bullseye: [python3-colorcet]
+  fedora: [python3-colorcet]
+  ubuntu:
+    '*': [python3-colorcet]
+    focal:
+      pip:
+        packages: [colorcet]
```
[the remark this answers is not in the document]
Comment (hsgwa) [reply rosdep/python.yaml:6195]: addressed in: [5499983](https://github.com/ros/rosdistro/pull/33681/commits/5499983069335724a294ce8ecbc51f87eddb47fd)

File: rosdep/python.yaml [hunk 3]
```diff
@@ -6181,6 +6181,15 @@ python3-colorama:
   openembedded: [python3-colorama@meta-python]
   rhel: ['python%{python3_pkgversion}-colorama']
   ubuntu: [python3-colorama]
+python3-colorcet:
+  debian:
+    bullseye: [python3-colorcet]
```
[the remark this answers is not in the document]
Comment (hsgwa) [reply rosdep/python.yaml]: addressed in: [5499983](https://github.com/ros/rosdistro/pull/33681/commits/5499983069335724a294ce8ecbc51f87eddb47fd)

[the remark this answers is not in the document]
Comment (hsgwa) [reply rosdep/python.yaml]: addressed in: 8cb4dfe

**Review by ivanpauno — APPROVED** [review]

**Pull request merged**